src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.sln

```diff
@@ -19,6 +19,14 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AirshipBindings.iOS.Extende
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AirshipBindings.iOS.MessageCenter", "..\AirshipBindings.iOS.MessageCenter\AirshipBindings.iOS.MessageCenter.csproj", "{AC22EEC5-DCA6-49D3-8B37-533B5565E2CF}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AirshipBindings.Android.HMS", "..\AirshipBindings.Android.HMS\AirshipBindings.Android.HMS.csproj", "{9ABA0E47-0B1A-4EC5-88D8-85E1E620E873}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AirshipBindings.Android.MessageCenter", "..\AirshipBindings.Android.MessageCenter\AirshipBindings.Android.MessageCenter.csproj", "{986507BE-5C63-4E62-B41E-7BE3FA2DB7C3}"

```

Comment:
Should HMS be included here? I think the apps add the push providers separately.